includes/class-newspack-newsletters-editor.php

```diff
@@ -126,6 +127,18 @@ public static function strip_editor_modifications() {
 		remove_editor_styles();
 		add_theme_support( 'editor-gradient-presets', array() );
 		add_theme_support( 'disable-custom-gradients' );
+		unregister_block_pattern( 'core/social-links-shared-background-color' );
+	}
+
+	/**
+	 * Remove Core's Remote Block patterns.
```

Comment:
```suggestion
	 * Remove Core's Remote Block patterns.
	 *
	 * @param boolean $should_load_remote Whether to load remote block patterns.
	 *
	 * @return boolean Whether to load remote block patterns.
```

Reply:
Fixed in https://github.com/Automattic/newspack-newsletters/pull/731/commits/ad94af1a867c4c706c749ab33c122c5d92aecfe4